usr.bin/vmstat/vmstat.c

```diff
@@ -1443,11 +1448,21 @@ domemstat_malloc(void)
 		    memstat_get_count(mtp) == 0)
 			continue;
 		xo_open_instance("memory");
+#if __has_feature(capabilities)
+		xo_emit("{k:type/%13s/%s} {:in-use/%5ju} "
+		    "{:memory-use/%5ju}{U:K} {:high-use/%7s} "
+		    "{:requests/%8ju} {:reservation-use/%8ju}{U:K}  ",
+		    memstat_get_name(mtp), (uintmax_t)memstat_get_count(mtp),
+		    ((uintmax_t)memstat_get_bytes(mtp) + 1023) / 1024, "-",
+		    (uintmax_t)memstat_get_numallocs(mtp),
+		    ((uintmax_t)memstat_get_reserved_bytes(mtp) + 1023) / 1024);
```

Comment:
Is it nicer to duplicate or shove #if's in the middle of the statement and reuse code?

Reply:
It feels bad either way, I do not have a specific preference. Maybe `#if` creates less diff, I should go with it.

Reply:
I tried avoiding the duplication but IMO it becomes worse, I'd leave this in.